File: python/7 kyu/135.py
# https://www.codewars.com/kata/5a1a9e5032b8b98477000004/train/python

# Passed

def even_last(numbers): 
    result = 0
    last_idx = len(numbers) - 1
    for idx, num in enumerate(numbers):
        if idx % 2 == 0:
            result += num

        if idx == last_idx:
            result *= num

    return result

output = even_last([2, 3, 4, 5])
print(output)

File: nodejs/7 kyu/135.js
// https://www.codewars.com/kata/563b1f55a5f2079dc100008a/train/javascript

// Passed

function getLargerNumbers(a, b) {
  let comparedArray = a.map((num, idx) => (num >= b[idx] ? num : b[idx]));
  return comparedArray;
}

const output = getLargerNumbers([13, 64, 15, 17, 88], [23, 14, 53, 17, 80]);
console.log(output);